presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -2,14 +2,16 @@
 Geospatial Functions
```

Comment:
This commit seems unnecessary. The current text has reasonable length. As long as it's in the 80-100 character range, it's fine.

Reply:
dropped